md/particle_interaction.py

```diff
@@ -555,7 +601,7 @@ def compute_forces(self,Positions,Sigma, Epsilon, Labels,L, switch_parameter, r_
             Positions_Difference[:,2] = Positions_Difference[:,2]%(L[2]/2)
             for j in neighbours[i]:
                 #Calculate the Distances 
-                dist = np.linalg.norm(Positions_Difference[j,:])
+                dist = np.linalg.norm(Positions_Difference[j,:])  #COMMENT (@skieninger): I think you can use the absolut distances we get from the neighbour list. dist == distances[i][j]
```

Comment:
thats My Part, 
I Agree that Dists could be importesd and used AT that Point,  but I need Positions_difference below